Tasks/KubernetesManifestV0/src/utils/DeploymentHelper.ts

```diff
@@ -92,6 +96,18 @@ async function checkManifestStability(kubectl: Kubectl, resources: Resource[]):
                 tl.warning(tl.loc('CouldNotDeterminePodStatus', JSON.stringify(ex)));
             }
         }
+        if (isEqual(resource.type, constants.DiscoveryAndLoadBalancerResource.service, StringComparer.OrdinalIgnoreCase)) {
+            try {
+                const service = getService(kubectl, resource.name);
+                const spec = service.spec;
+                const status = service.status;
+                if (isEqual(spec.type, "LoadBalancer", StringComparer.OrdinalIgnoreCase) && !isLoadBalancerIPAssigned(status)) {
```

Comment:
Use `loadbalancer` from `DiscoveryAndLoadBalancer` constants.

Reply:
added ServiceTypes for this.